test/kubemark/start-kubemark.sh

```diff
@@ -167,18 +167,7 @@ function create-and-upload-hollow-node-image {
   CURR_DIR=`pwd`
```

Comment:
Even this logic of changing directory, etc should be moved into make-kubemark-container function or you should explicitly add a comment above the function skeleton saying that it is assumed that this function runs while we are in the `cluster/kubemark/images` directory. However, the preferable solution would be neither of these, but the one in my comment below.